contributing/pull_request.md

```diff
@@ -2,25 +2,23 @@
 
 ## Contributions
 
-We expect all contributions to
-- conform to our [style guide](./styleguide.md),
-- be commented (inside the PHP source files),
-- be documented (in the [user guide](https://codeigniter4.github.io/userguide/)),
-- and unit tested (in the [test folder](https://github.com/codeigniter4/CodeIgniter4/tree/develop/tests)).
+We expect the following in all pull requests.
+- All git commits are [GPG-signed](#signing)
+- Must follow our [style guide](#php-style)
+- Be [commented](#comments) in the PHP source file
+- Be documented in the [user guide](#user-guide)
+- Be [unit tested](#unit-testing)
+- Pass all checks in GitHub Actions
 
```

Comment:
Emphasis on sending PRs to the appropriate branch (fixing bugs to develop, etc.) might be good.

Reply:
Good! Done.